src/engine/Settings.cs

```diff
@@ -185,6 +185,8 @@ private Settings()
     /// </summary>
     public SettingValue<string> CustomUsername { get; set; } = new SettingValue<string>(null);
 
+    public SettingValue<bool> OpenScreenShotFolder { get; set; } = new SettingValue<bool>(false);
```

Comment:
Why did you add a new setting here? A button just for opening the screenshot folder doesn't require any state?